README.md

```diff
@@ -10,50 +10,89 @@ Currently [Mozilla is considering implementation of the feature](https://github.
 
 ## Use case
 
-The constructible style sheets proposal is intended to allow for the dynamic creation and sharing of style sheets, even across shadow boundaries. By adopting a style sheet into a shadow root, the same sheet can be applied to multiple nodes, including the document. 
+The constructible style sheets proposal is intended to allow for the dynamic
+creation and sharing of style sheets, even across shadow boundaries. By adopting
+a style sheet into a shadow root, the same sheet can be applied to multiple
+nodes, including the document.
 
 ## How it works
 
-This polyfill will create a new style element for every `DocumentOrShadowRoot` into which the sheet is adopted. This is counter to the current proposal, but updates to the style sheet using the `replace` or `replaceSync` methods should update the relevant style elements with the updated content across all adopters.
+This polyfill will create a new style element for every `DocumentOrShadowRoot`
+into which the sheet is adopted. This is counter to the current proposal, but
+updates to the style sheet using the `replace` or `replaceSync` methods should
+update the relevant style elements with the updated content across all adopters.
 
 No changes will occur in a browser that supports the feature by default.
 
-## Requirements
+## Support
 
-This polyfill should work as-is in modern browsers, however, to support IE11, you will also need to consume a polyfill for [ECMAScript `Symbol`s](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol). One popular and well supported option is [Core-JS](https://www.npmjs.com/package/core-js) which includes polyfills for many other modern features alongside a polyfill for `Symbol`.
+This polyfill supports all modern browsers and IE 11. No additional polyfills
+required.
+
+For browsers that do not support the web components specification (currently
+IE 11 and Edge) only the document-level style sheets adoption works. 
```

Comment:
And checking `instanceof` will not work in IE11 due to `Symbol` not being natively supported.

Reply:
Hmm, I just fixed it and the previous one. It looks the following now:
```html
<script src="../dist/adoptedStyleSheets.js"></script>
```
and 
```markdown
## Support
This polyfill supports all modern browsers and IE 11.

For browsers that do not support the web components specification (currently
IE 11 and Edge) only the document-level style sheets adoption works.

### IE 11

To make this polyfill work with IE 11 you need the following tools:
- `Symbol` polyfill (with support for `Symbol.hasInstance`).
- [@babel/plugin-transform-instanceof](https://www.npmjs.com/package/@babel/plugin-transform-instanceof)
applied to your code that uses `instanceof` against `CSSStyleSheet`.
```